src/main/java/io/odpf/firehose/consumer/kafka/FirehoseKafkaConsumer.java

```diff
@@ -17,9 +17,7 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.stream.Collectors;
 
-import static io.odpf.firehose.metrics.Metrics.FAILURE_TAG;
-import static io.odpf.firehose.metrics.Metrics.SOURCE_KAFKA_MESSAGES_COMMIT_TOTAL;
-import static io.odpf.firehose.metrics.Metrics.SUCCESS_TAG;
+import static io.odpf.firehose.metrics.Metrics.*;
```

Comment:
don't put asterisk. It will fail the checkstyle..expand it like it was before

Reply:
```
        <module name="AvoidStarImport">
          <property name="allowStaticMemberImports" value="true"/>
        </module>
```
it passed the checkstyle because currently the star (.*) import is currently allowed for static import
should we changes the checkstyle config too ?

Reply:
Yes. that's fine. 
but you don't have to edit current checkstyle or the code.

Reply:
you can add your import there, don't delete the previous lines.

Reply:
changes added